psh/top/top.c

```diff
@@ -191,7 +191,7 @@ static void psh_top_refresh(char cmd, threadinfo_t *info, threadinfo_t *previnfo
 
 	/* Set header style */
 	printf("\033[0;30;47m");
-	printf("%5s %5s %2s %5s %5s %5s %9s %8s %-28s\n", "PID", "PPID", "PR", "STATE", "%CPU", "WAIT", "TIME", "VMEM", "CMD");
+	printf("%8s %8s %2s %5s %5s %6s %9s %8s %-21s\n", (psh_top_common.threads) ? "TID" : "PID", "PPID", "PR", "STATE", "%CPU", "WAIT", "TIME", "VMEM", "CMD");
```

Comment:
I think PID should be displayed in either case. I guess we cannot fit that many fields?

Reply:
That's right - adding PID while displaying threads would make very little space for the CMD.